pkg/create/prompts.go

```diff
@@ -0,0 +1,90 @@
+package create
+
+import (
+	"fmt"
+	"regexp"
+	"slices"
+
+	"github.com/AlecAivazis/survey/v2"
```

Comment:
The `survey/v2` library is archived (since April 2024) and unmaintained. The CLI already uses `pterm` for interactive prompts (see `browsers.go`, `profiles.go`). Consider using:
- `pterm.DefaultInteractiveTextInput` for text input  
- `pterm.DefaultInteractiveSelect` for selection menus

This keeps the codebase consistent and avoids adding an archived dependency.